packages/reason/package.json

```diff
@@ -22,7 +26,8 @@
   ],
   "description": "Phenomic Reason bindings",
   "dependencies": {
-    "reason-react": "^0.2.3"
+    "reason-react": "^0.2.3",
+    "@phenomic/plugin-renderer-react": "^1.0.0-alpha.4"
```

Comment:
I'll rename reason to reasonReact in case people want to use Tea or alternatives